prime-router/metadata/fhir_mapping/hl7/datatypes/XTN/ContactPoint.yml

```diff
@@ -0,0 +1,290 @@
+value_1:
+  condition: $telEquipType NOT_EQUALS Internet && $telEquipType NOT_EQUALS X.400 && $localNum NULL && $unformattedTelNum NULL && $telNum NOT_NULL
+  valueOf: $telNum
+  vars:
+    telNum: String, XTN.1
+    telEquipType: String, XTN.3
+    localNum: String, XTN.7
+    unformattedTelNum: String, XTN.12
+
+value_2:
+  condition: $telEquipType NULL && $localNum NULL && $unformattedTelNum NULL && $telNum NOT_NULL
+  valueOf: $telNum
+  vars:
+    telNum: String, XTN.1
+    telEquipType: String, XTN.3
+    localNum: String, XTN.7
+    unformattedTelNum: String, XTN.12
+
+value_3:
+  condition: $telEquipType EQUALS Internet || $telEquipType EQUALS X.400
+  valueOf: $commAddress
+  vars:
+    telEquipType: String, XTN.3
+    commAddress: String, XTN.4
+
+value_4:
+  condition: $telEquipType NOT_EQUALS Internet && $telEquipType NOT_EQUALS X.400 && $unformattedTelNum NOT_NULL
+  valueOf: unformattedTelNum
+  vars:
+    telEquipType: String, XTN.3
+    unformattedTelNum: String, XTN.12
+
+use_1:
+  condition: $useCode EQUALS PRN
+  type: STRING
+  value: "home"
+  vars:
+    useCode: String, XTN.2
+
+use_2:
+  condition: $useCode EQUALS WPN
+  type: STRING
+  value: "work"
+  vars:
+    useCode: String, XTN.2
+
+use_3:
+  condition: $useCode EQUALS BPN
+  type: STRING
+  value: "mobile"
+  vars:
+    useCode: String, XTN.2
+
+#from the PID[Patient] mapping inventory
+use_4:
+  condition: $useCode NULL
+  type: STRING
+  value: "home"
+  vars:
+    useCode: String, XTN.2
+
+system_1:
+  condition: $equipType EQUALS PH
+  type: STRING
+  valueOf: "phone"
+  vars:
+    equipType: String, XTN.3
+
+system_2:
+  condition: $equipType EQUALS FX
+  type: STRING
+  valueOf: "fax"
+  vars:
+    equipType: String, XTN.3
+
+system_3:
+  condition: $equipType EQUALS Internet || $equipType EQUALS X.400
+  type: STRING
+  valueOf: "email"
+  vars:
+    equipType: String, XTN.3
+
+system_4:
+  condition: $equipType EQUALS BP
+  type: STRING
+  valueOf: "pager"
+  vars:
+    equipType: String, XTN.3
+
+system_5:
+  condition: $equipType EQUALS MD || $equipType EQUALS SAT || $equipType EQUALS TDD || $equipType EQUALS TTY
+  type: STRING
+  valueOf: "other"
+  vars:
+    equipType: String, XTN.3
+
+system_6:
+  condition: $telEquipType NULL && $commAddress NOT_NULL
+  valueOf: "email"
+  vars:
+    telEquipType: String, XTN.3
+    commAddress: String, XTN.4
+
+system_7:
+  condition: $telEquipType NULL && $commAddress NULL
+  valueOf: extension/DataAbsentReason
+  expressionType: resource
+  vars:
+    telEquipType: String, XTN.3
+    commAddress: String, XTN.4
+
+extension_1-telecom-equipment-type:
+  condition: $equipType NOT_NULL
```

Comment:
these conditions are not always needed since setting a variable. But I think having them makes the intension more clear